roles/confluent.variables/defaults/main.yml

```diff
@@ -494,10 +516,10 @@ kafka_connect_monitoring_interceptors_enabled: "{{ monitoring_interceptors_enabl
 
 #### KSQLDB Variables ####
 
-### Only use to customize Linux User ksqlDB Service runs with. User must exist on host.
+### Only use to customize Linux User ksqlDB Service runs with. Default user is cp-ksql
 ksql_user: "{{ksql_default_user}}"
 
-### Only use to customize Linux Group ksqlDB Service user belongs to. Group must exist on host.
+### Only use to customize Linux Group ksqlDB Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the ksqlDB Service user belongs to. Default group is confluent.